docs/source/envvars.rst

```diff
@@ -1,15 +1,501 @@
+
+
+IoT agent MQTT
+--------------
+
+.. code-block:: bash
+
+    # REDIS instance for Mosca backend
+    export BACKEND_HOST="mosca-redis"
+    export BACKEND_PORT=6379
+
+    # Enable or disable unsecured mode
+    # Encrypted communication is always enabled.
+    export ALLOW_UNSECURED_MODE="false"
+
+    # Log level. Accepted values: info, warn, error and debug.
+    export LOG_LEVEL="info"
+
+    # Mosca TLS configuration
+    export MOSCA_TLS_SECURE_CERT="/opt/iot-agent/mosca/certs/mosca.crt"
+    export MOSCA_TLS_SECURE_KEY="/opt/iot-agent/mosca/certs/mosca.key"
+    export MOSCA_TLS_CA_CERT="/opt/iot-agent/mosca/certs/ca.crt"
+    export MOSCA_TLS_DNS_LIST="mqtt,mosca,localhost"
+
+    # IoT agent healthcheck configuration
+    # All values are in miliseconds and indicate how much time it will
+    # wait before checking a particular state again.
+    export HC_UPTIME_TIMEOUT=300000
+    export HC_MEMORY_USAGE_TIMEOUT=300000
+    export HC_CPU_USAGE_TIMEOUT=300000
+    export HC_MONGODB_TIMEOUT=30000
+    export HC_KAFKA_TIMEOUT=30000
+
+    #
+    # dojot-module related variables
+    #
+
+    # Comma separated list of Kafka instances to be used for Kafka client
+    # bootstrapping.
+    export KAFKA_HOSTS="kafka:9092"
+    # Kafka group ID for consumers. It is advisible that this is the same
+    # for instances of the same service and different for each service.
+    export KAFKA_GROUP_ID="kafka"
+    # Amount of time (in seconds) that the service will wait before attempting
+    # to subscribe to a set of Kafka topics.
+    export DOJOT_KAFKA_SUBSCRIPTION_HOLDOFF="2"
+    # Time between each poll
+    export DOJOT_KAFKA_POLL_TIMEOUT="2000"
+    # Where Data Broker is
+    export DATA_BROKER_URL="http://data-broker"
+    # Where Device Manager is
+    export DEVICE_MANAGER_URL="http://device-manager:5000"
+    # Where Auth is
+    export AUTH_URL="http://auth:5000"
+
+    # Tenant used for management operations and associated messages, such as
+    # tenant creation
+    export DOJOT_MANAGEMENT_TENANT="dojot-management"
+    # User for management request, such as requesting the list of currently
+    # configured tenants.
+    export DOJOT_MANAGEMENT_USER="dojot-management"
+    # Subject used for tenancy messages, such as tenant CRUD messages
+    export DOJOT_SUBJECT_TENANCY="dojot.tenancy"
+    # Subject used for device management messages, such as creation.
+    export DOJOT_SUBJECT_DEVICES="dojot.device-manager.device"
+    # Subject used for device messages, such as device attributes update
+    export DOJOT_SUBJECT_DEVICE_DATA="device-data"
+
+
 Data Broker
 -----------
 
 .. code-block:: bash
 
+    # Redis instance used by Data Broker
+    export DATABROKER_CACHE_HOST="data-broker-redis"
+
+    # Kafka configuration. These are used for topic creation.
+    export DATABROKER_KAFKA_ADDRESS="kafka"
+    export DATABROKER_KAFKA_PORT=9092
+    export DATABROKER_ZOOKEEPER_HOST="zookeeper:2181"
```

Comment:
I didn't find anything in Data Broker code that uses the variable DATABROKER_ZOOKEEPER_HOST. I am removing it from Data Broker code and documentation, we should remove it from here too.